src/sections/Terms/index.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+
+import { Container, Row, Col } from "../../reusecore/Layout";
+import PageHeader from "../../reusecore/PageHeader"
+import SectionTitle from "../../reusecore/SectionTitle"
```

Comment:
Remove this

Reply:
But Container Row and Col are being used by it provides a layout similar to other pages and sections

Reply:
I meant only the `SectionTitile`

Reply:
OK